src/Hyperplane.jl

```diff
@@ -395,3 +395,8 @@ function _constraints_list_hyperplane(a::AbstractVector{N}, b::N
                                      )::Vector{LinearConstraint{N}} where {N<:Real}
     return [HalfSpace(a, b), HalfSpace(-a, -b)]
 end
+
+function _linear_map_hrep(M::AbstractMatrix{N}, P::Hyperplane{N}, use_inv::Bool) where {N<:Real}
+    constraint = _linear_map_hrep_helper(M, P, use_inv)
+    return Hyperplane(constraint[1].a, constraint[1].b)
```

Comment:
Hm, so for the linear map with the identity matrix we might change the `b` value to `-b`. I think that is fine, but I wanted to mention it.

Reply:
True. At first i thought about adding some "normaization" but since the two forms `ax = b`  and `-ax = -b` are mathematically equivalent, I think that it is acceptable.

Reply:
Yes, the sets are the same. It may just be unexpected.